lib/refresh.fish

```diff
@@ -4,6 +4,9 @@
 # OVERVIEW
 #   Refresh (reload) the current fish session.
```

Comment:
Refresh or reload? This comment is so weird... if `reload` is easier shouldn't we just call this function `reload`? :trollface: